runtime/onert/backend/cpu/StaticTensorManager.cc

```diff
@@ -97,8 +97,6 @@ void StaticTensorManager::iterate(const std::function<void(const ir::OperandInde
 {
   for (const auto &it : _tensors->managed_tensors())
```

Comment:
This line is not part of this PR, but it seems that this line should be `_tensors->native_tensors()` since `MigrantTensor` becomes `NativeTensor`.

Reply:
I'll do that, thanks.